images/macos/macos-14-arm64-Readme.md

```diff
@@ -205,16 +204,16 @@
 | Simulator - watchOS 10.5 | watchsimulator10.5   | 15.4          |
 | Simulator - watchOS 11.1 | watchsimulator11.1   | 16.1          |
 | Simulator - watchOS 11.2 | watchsimulator11.2   | 16.2          |
-| Simulator - visionOS 1.0 | xrsimulator1.0       | 15.2          |
 | visionOS 1.0             | xros1.0              | 15.2          |
-| Simulator - visionOS 1.1 | xrsimulator1.1       | 15.3          |
 | visionOS 1.1             | xros1.1              | 15.3          |
 | visionOS 1.2             | xros1.2              | 15.4          |
-| Simulator - visionOS 1.2 | xrsimulator1.2       | 15.4          |
 | visionOS 2.1             | xros2.1              | 16.1          |
+| visionOS 2.2             | xros2.2              | 16.2          |
+| Simulator - visionOS 1.0 | xrsimulator1.0       | 15.2          |
```

Comment:
Curious why these rows gets reshuffled? Not a big deal but it makes it harder to review changes. Do we sort records in particular order to make results deterministic?

Reply:
Yeah, I enabled sorting for visionOS with one of the latest PRs, so with the next releases it's expected to become more stable.

Reply:
What's much more interesting is what happened to the next block. 😄